src/app/app.module.ts

```diff
@@ -14,7 +17,10 @@ import { AppComponent } from './app.component';
     BrowserAnimationsModule,
     FormEntryModule,
     HttpClientModule,
-    ReactiveFormsModule
+    ReactiveFormsModule,
+    TranslateModule.forRoot({
```

Comment:
Should the loader also be set up? How would one supply these / custom translations?

Reply:
The translations seem to be coming from the backend, so I don't know to be honest.

Reply:
I am actually going to add the loader, I have been reading through some tutorial and I see that it is also supposed to be added. Thanks @FlorianRappl

Reply:
Example with custom deps to determine the base path (something we'd also need to think about, as usage in micro frontends implies usage with different base path):

```
// loader
function HttpLoaderFactory(http: HttpClient, piral: PiletApi) {
  return new TranslateHttpLoader(http, `${piral.meta.basePath}i18n/`, '.json');
}

// in the config
TranslateModule.forRoot({
      defaultLanguage: 'en',
      loader: {
        provide: TranslateLoader,
        useFactory: HttpLoaderFactory,
        deps: [HttpClient, 'piral'],
      },
    }),
```

Hope that helps!

(It's / will be tricky)

Reply:
Actually, loading the translations files is probably functionality we should have in the @openmrs/esm-form-entry-app, rather than here. From that perspective, I'm not even sure we need or want the http-loader as part of this repo. (This is similar to how we handle the form itself IIRC). What we need here is to ensure that all the labels, placeholders, etc. can be translated.

@hadijahkyampeire Do you have some screenshots showing this?

Reply:
I am going to share them ASAP.